django/db/models/aggregates.py

```diff
@@ -65,7 +65,9 @@ def resolve_expression(self, query=None, allow_joins=True, reuse=None, summarize
         if hasattr(default, 'resolve_expression'):
             default = default.resolve_expression(query, allow_joins, reuse, summarize)
         c.default = None  # Reset the default argument before wrapping.
-        return Coalesce(c, default, output_field=c._output_field_or_none)
+        coalesce = Coalesce(c, default, output_field=c._output_field_or_none)
+        coalesce.is_summary = True
```

Comment:
Actually, shouldn't this be

```suggestion
        coalesce.is_summary = summarize
```

Reply:
Seems cromulent. Is there a test we could add for that case?

Reply:
Agreed, I just need to figure out how to test it.

Reply:
No luck so far  :confused: Suggestions are welcome.

Reply:
Added :rocket: `test_aggregation_default_not_in_aggregate()` crashes with:
```
django.db.utils.ProgrammingError: column "subquery.__col1" must appear in the GROUP BY clause or be used in an aggregate function
LINE 1: SELECT COALESCE("__col1", 2.5), SUM("__col2") FROM (SELECT A...
```
without using `c.is_summary`.